packages/features/insights/components/HighestNoShowHostTable.tsx

```diff
@@ -0,0 +1,47 @@
+import { Title } from "@tremor/react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc";
+
+import { useFilterContext } from "../context/provider";
+import { CardInsights } from "./Card";
+import { LoadingInsight } from "./LoadingInsights";
+import { TotalUserFeedbackTable } from "./TotalUserFeedbackTable";
+
+export const HighestNoShowHostTable = () => {
+  const { t } = useLocale();
+  const { filter } = useFilterContext();
+  const { dateRange, selectedEventTypeId, isAll, initialConfig } = filter;
+  const [startDate, endDate] = dateRange;
+  const { selectedTeamId: teamId } = filter;
+
+  const { data, isSuccess, isPending } = trpc.viewer.insights.membersWithMostNoShow.useQuery(
+    {
+      startDate: startDate.toISOString(),
+      endDate: endDate.toISOString(),
+      teamId,
+      eventTypeId: selectedEventTypeId ?? undefined,
+      isAll,
+    },
+    {
+      staleTime: 30000,
+      trpc: {
+        context: { skipBatch: true },
+      },
+      enabled: !!(initialConfig?.teamId || initialConfig?.userId || initialConfig?.isAll),
```

Comment:
The earlier de-structuring makes me suspect this doesn't work, needs clarification.